app/controllers/orders/client_summary.js

```diff
@@ -6,6 +6,22 @@ const { getOwner } = Ember;
 export default detail.extend({
   showBeneficiaryModal: false,
 
+  titles: Ember.computed(function(){
+    return [
+      { name: "Mr", id: "Mr" },
+      { name: "Mrs", id: "Mrs" },
+      { name: "Miss", id: "Miss" },
+      { name: "Ms", id: "Ms" }
+    ];
+  }),
+
+  identityTypes: Ember.computed(function() {
```

Comment:
Would it be better to add model and fetch `identityType`?  Just a Thought.

Reply:
It will only return one identityType and we need both identityType for Select list so  :)